test/fixtures/poro.rb

```diff
@@ -3,23 +3,19 @@
 class Model < ActiveModelSerializers::Model
   FILE_DIGEST = Digest::MD5.hexdigest(File.open(__FILE__).read)
 
-  ### Helper methods, not required to be serializable
-
-  # Convenience when not adding @attributes readers and writers
-  def method_missing(meth, *args)
-    if meth.to_s =~ /^(.*)=$/
-      attributes[Regexp.last_match(1).to_sym] = args[0]
-    elsif attributes.key?(meth)
-      attributes[meth]
-    else
-      super
-    end
+  # At this time, just for organization of intent
```

Comment:
probably good to promote to parent class once we add some additional behavior to the `attributes` method